spec/features/staff/users_can_upload_activities_spec.rb

```diff
@@ -207,6 +226,19 @@
     end
   end
 
+  def expect_change_to_be_recorded_as_historical_event(field:, previous_value:, new_value:, activity:)
+    historical_event = HistoricalEvent.find_by!(value_changed: field)
+
+    aggregate_failures do
```

Comment:
First time I have seen this as well! Very nice, I had to look it up! 

This makes rspec run all the expectations in the block even if one fails, is that right?

Reply:
yes this is handy for those specs where it makes sense to break the 'one assertion per test' convention...